src/Controller/AdminController.php

```diff
@@ -506,4 +513,38 @@ public function rescheduleNotification(string $notificationId): JsonResponse
 
         return new JsonResponse($notificationId);
     }
+
+    /**
+     * @param string $pluginId
+     * @return JsonResponse
+     */
+    #[Route(
+        '/api/adyen/orders/{orderId}/is-adyen-order',
+        name: 'api.adyen_is_adyen_order.get',
+        methods: ['GET']
+    )]
+    public function isAdyenOrder(string $orderId): JsonResponse
+    {
+        try {
+            $adyenPluginId = $this->pluginProvider->getAdyenPluginId();
+            $transaction = $this->orderTransactionRepository->getFirstAdyenOrderTransaction($orderId, Context::createDefaultContext());
+
+            if (!is_null($transaction)) {
+                $pluginId = $transaction->getPaymentMethod()->getPluginId();
+
+                if($pluginId == $adyenPluginId){
```

Comment:
This check is redundant since `getFirstAdyenOrderTransaction()` method has already had a `plugin` filter [here](https://github.com/Adyen/adyen-shopware6/blob/562275cc1ce4fe875086126c6ff253f4b3744d09/src/Service/Repository/OrderTransactionRepository.php#L94) for Adyen payments. Obtaining `transaction` is sufficient since the criteria filters only Adyen order transactions.